src/components/common/ButtonBase.vue

```diff
@@ -0,0 +1,23 @@
+<script>
+import { defineComponent } from 'vue';
+export default defineComponent({
+});
+</script>
+
+<template>
+  <button class="button">
+    <slot></slot>
+  </button>
+</template>
+
+<style lang="scss" scoped>
+.button {
+  height: 40px;
+  width: 100px;
```

Comment:
I would really avoid fixed width's on things like buttons - we do not know how big the button would be, it's contents would change the size.

Reply:
This case perfectly shows, why there should be one versatile button component! 😄 I created ButtonBase as a sandbox for my first trials with Vue and then left it and continued styling only ButtonlikeLink as this was what I needed for my landing page. Sorry for these working code residues in the PR... 😕

Reply:
No problem! Now I got it 😄